### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by stipot** [Aeraphe/xvba-cli#2]
Using npm i @localsmart/xvba-cli@latest --save-dev
I've got error: Not Found - GET https://registry.npmjs.org/excel-types,  'excel-types@1.0.0' is not in this registry.

**Comment by yves-amevoin** [comment]
Hi @stipot,
Same for me. Please, did you found a way to get arround this ?

**Comment by stipot** [comment]
> Hi @stipot, Same for me. Please, did you found a way to get arround this ?
No, I didn't((

**Comment by stipot** [comment]
> Hi @stipot, Same for me. Please, did you found a way to get arround this ?
I used VBA - Live server .. exention for VS Code.
And I found that .vb file extention work well with autocomplete and comment blocks. But it doesn't work with .bas extention. I cluldn't fuguere out how to make it avalable for the .bas exention.
I use a local fix by the VS Code extention "Run on save". So I work in .vb files and on save event the extention "Run on save" just copy it to *.bas files which then could be exported to the Excel.
dumb but works
